packages/animations/CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
-## [1.0.1-dev] - TBD
+# Changelog
 
+All notable changes to this project will be documented in this file.
+
+## [1.1.1-dev] - TBD
+
+* Update with any notable changes since the last release here.
```

Comment:
This will show up on https://pub.dev/packages/animations#-changelog-tab-, which is probably not desirable. We should probably leave this out and need to remember to add it back in when the first dev commit lands (or add it in a separate PR right after this one is published to pub).